chart/kubeapps/templates/apprepositories.yaml

```diff
@@ -4,19 +4,7 @@ kind: AppRepository
 metadata:
   name: {{ .name }}
   annotations:
-{{- if semverCompare ">=2.14.0" $.Capabilities.TillerVersion.SemVer }}
```

Comment:
@andresmgot Do we need to continue the pre 2.14 support? If so, I'll dig more to ensure this condition can still work if `$.Capabilities.TillerVersion` is defined and be ignored otherwise.

Reply:
The first version of 2.14 was released on May so I guess there will be people still using 2.13. There may be a reason for people to still use 2.13 so I would maintain this if possible.

Reply:
> The first version of 2.14 was released in May so I guess there will be people still using 2.13. There may be a reason for people to still use 2.13 so I would maintain this if possible.

So i took a look at the history here. AFAICT (from https://github.com/kubeapps/kubeapps/pull/955#pullrequestreview-200764260 ), the issue is that `pre-install` didn't work with helm 2.11 (see your comment there where both you and Adnan are unable to reproduce with 2.12 but could with 2.11), which is a year old now. It was just that the work-around of using the `crd-install` hook stopped working with 2.13 for non-crd's (the app repos).

So we'd be maintaining this conditional for a year old release. I don't see any way with the built in template vars to determine whether the helm version is > 2.11 moving forward, as the capabilities are only the [kube version and api info now](https://github.com/helm/helm/blob/v3.0.0-beta.3/pkg/chartutil/capabilities.go#L37). So unless I'm missing some other built in that tells us the running tiller version, or we patch helm upstream leaving the tiller version field there but setting to zero or similar, we'd need to make the condition based on whether the field exists, which is [pretty strongly discouraged](https://stackoverflow.com/a/44676179) and requires a custom template function in the template context. All of which leaves me -1 to include the work-around, and +1 to recommend people use a version later than helm 2.11 if this is an issue. I hope that's OK (assuming so, based on your previous approval).

Reply:
Thanks for the investigation. It's fine then